### DISCUSSION TITLE
Batching with multiple graphs

### DISCUSSION THREAD

**Discussion by mcunningham1440** [pyg-team/pytorch_geometric#8180] [Q&A] [answered]
I have a dataset in which each example is a list of heterogeneous graphs associated with a numeric label, e.g. X = [graph1, graph2, graph3, graph4, graph5] and y = 1.2345. I would like to use the PyG dataloader to feed them into a model, but obviously this requires loading multiple graphs per item

The Advanced Mini-Batching article lays out how to do this for a pair of graphs, but not more than this. Is this possible to do, and, if so, how should I go about it? Thanks!

**Comment by akihironitta** [comment, marked as answer]
Your use case sounds very similar to #7928. Have you had a chance to try https://github.com/pyg-team/pytorch_geometric/discussions/7928#discussioncomment-6820073?

**Reply by mcunningham1440** [reply]
Apologies, I searched through the previous comments but didn't find that one. Thanks for directing me to it--yes, it solves my problem
